tarantool/connection.py

```diff
@@ -363,7 +363,8 @@ def __init__(self, host, port,
                  ssl_ca_file=DEFAULT_SSL_CA_FILE,
                  ssl_ciphers=DEFAULT_SSL_CIPHERS,
                  packer_factory=default_packer_factory,
-                 unpacker_factory=default_unpacker_factory):
+                 unpacker_factory=default_unpacker_factory,
+                 fetch_schema=True):
```

Comment:
Let's also use `fetch_schema=False` connection in crud API example.

Reply:
Done. Also now `fetch_schema=False` used in CRUD tests